src/operation/decision_model.jl

```diff
@@ -271,6 +277,33 @@ end
 
 function validate_time_series(model::DecisionModel{<:DefaultDecisionProblem})
     sys = get_system(model)
+    settings = get_settings(model)
+    available_resolutions = PSY.list_time_series_resolutions(sys)
+
+    if get_resolution(settings) == UNSET_RESOLUTION && length(available_resolutions) != 1
```

Comment:
This logic is repeated and could easily be put in a function.

Reply:
agreed